### PR TITLE
Minor changes during pipeline setup

### PR DESCRIPTION
**Pull request by zhanggbj**
1.increase timeout to run integration test smoothly
2.format command line output
[#120520321](https://www.pivotaltracker.com/story/show/120520321)

### CODE REVIEW

File: integration/bmp/bms_cmd_test.go [hunk 1]
```diff
@@ -22,7 +21,7 @@ var _ = Describe("`$bmp bms --deployment` integration tests", func() {
 
 	Describe("$bmp bms --deployment", func() {
 		BeforeEach(func() {
-			deployment := GetDeployment()
+			deployment, _ := GetDeployment()
```
Comment (maximilien) [inline integration/bmp/bms_cmd_test.go]: Why are you ignoring the error? Please add a check for no error

**Comment by maximilien** [comment]
See comment. Please address and I will merge.

**Comment by zhanggbj** [comment]
Comments addressed. Thanks.

**Comment by maximilien** [comment]
Cool, merging. Cheers 👍

**Pull request merged**